samples/outbox/multi-tenant/Version_6/Shared/ConfigErrorQueue.cs

```diff
@@ -0,0 +1,13 @@
+using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+internal class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
+{
+    public MessageForwardingInCaseOfFaultConfig GetConfiguration()
+    {
+        return new MessageForwardingInCaseOfFaultConfig
+        {
+            ErrorQueue = "error"
```

Comment:
Sure, will do. Wanted to do a bit of 'house-keeping' but thought to leave it for now for people to review, as it is already different than v5.